user_guide_src/source/intro/requirements.rst

```diff
@@ -49,11 +49,11 @@ Supported Databases
 A database is required for most web application programming.
 Currently supported databases are:
 
-  - MySQL via the *MySQLi* driver (version 5.1 and above only)
-  - PostgreSQL via the *Postgre* driver
-  - SQLite3 via the *SQLite3* driver
-  - MSSQL via the *SQLSRV* driver (version 2005 and above only)
-  - Oracle via the *OCI8* driver (version 12.1 and above only)
+  - MySQL via the ``MySQLi`` driver (version 5.1 and above only)
+  - PostgreSQL via the ``Postgre`` driver
```

Comment:
```suggestion
  - PostgreSQL via the ``Postgre`` driver
```

Not consistent in configuration file, use `Postgres` or `Postgre`? I am not using PostgreSQL

Reply:
Good catch! It is the directory name for the driver. That is `Postgre`.